### PR TITLE
simplify install instructions

### PR DESCRIPTION
**Pull request by ahnitz**
The install instructions included a lot of outdated information that is confusing people. This cuts that out and I've put in the version I use personally for cluster / dev installs.

### CODE REVIEW

**Comment by ahnitz** [comment]
Here's what the page looks like 
https://www.atlas.aei.uni-hannover.de/work/ahnitz/projects/tplot/html/install.html

**Comment by josh-willis** [comment]
@ahnitz Isn't it still true that we need to run `pip install --upgrade pip setuptools` inside the virtualenv, before installing anything else?  If that's still the case for RHEL7 I would argue that we need to leave that in, but if not, then what you have now is pretty close to what I usually do myself.

**Comment by ahnitz** [comment]
@josh-willis I think that's a question for @spxiwh or someone else using a RHEL env. It's not needed in most modern OSs. I don't have a RHEL env on hand to check.

**Comment by josh-willis** [comment]
@ahnitz Do you remember what the specific minimal version of `setuptools` and `pip` that we need is? I can create a dummy virtualenv pretty easily to check that, but I just don't recall exactly what the problem was that we were needing to solve with that step.

**Comment by ahnitz** [comment]
@spxiwh Poke

File: docs/install.rst [hunk 1]
```diff
@@ -9,292 +9,60 @@ Simple Installation
...
 PyCBC is available through the PyPI. For straightforward use of the PyCBC library
-and executables, we recommend installing with the following command. If you 
+and executables, we recommend installing with the following command. If you
 are not running in a specialized computing environment, this is probably the
 appropriate thing to do.
 
 .. code-block:: bash
-    
+
     pip install lalsuite pycbc
```
Comment (spxiwh) [inline docs/install.rst:18]: Is this spacing not needed for the formatting in Sphinx? Can you check this still displays?

File: docs/install.rst [hunk 2]
```diff
@@ -9,292 +9,60 @@ Simple Installation
...
-    where branch_name is the name of the branch that you want to track.
-
-To build and install any changes that you make to the source code in your virtual environment, run the command
+Finally, install the most common pycbc develeopment environment packages
+as follows.
 
 .. code-block:: bash
 
+    cd pycbc
+    pip install -r requirements.txt
```
Comment (spxiwh) [inline docs/install.rst:56]: Can we also highly recommend users to install `companion.txt` as well, unless they need the diskspace.

File: docs/install.rst [hunk 3]
```diff
@@ -304,53 +72,24 @@ To build the documentation from your virtual environment, first make sure that y
...
-    directory you can delete the contents of your git repository. If you do
-    this by accident, you can use ``git reset`` to undo the commit.
-
-For more details on building and maintaining the documentation under GitHub project pages, see
-
-.. toctree::
-    :maxdepth: 1
-
-    build_gh_pages
+    python setup.py build_docs
```
Comment (spxiwh) [inline docs/install.rst:80]: Where will the documentation be?

**Comment by spxiwh** [comment]
@ahnitz As `pip` always suggests upgrading anyway, is it not good practice to just upgrade these as a first step anyway? I never seem to hit issues with this myself, but I suspect pip caches the newer versions and installs those when I create a new venv.

**Review by spxiwh — APPROVED** [review]
There's some comments, but I think this will make installation a lot clearer for the standard user.

File: docs/install.rst (hunk 1 above)
Comment (ahnitz) [reply docs/install.rst:18]: Only change is whitespace removal. The newline is still there.

Comment (ahnitz) [reply docs/install.rst:18]: Yes, this displays correctly.

File: docs/install.rst (hunk 2 above)
Comment (ahnitz) [reply docs/install.rst:56]: Ok, added that in.

**Comment by ahnitz** [comment]
@spxiwh Yes, I've also added in the recommendation to install the latest pip / setuptools.

File: docs/install.rst (hunk 3 above)
Comment (ahnitz) [reply docs/install.rst:80]: I've added a sentence to say where they will show up.

**Pull request merged**